class.jetpack.php

```diff
@@ -1098,8 +1098,10 @@ function user_role_change( $user_id ) {
 	 * Loads the currently active modules.
 	 */
 	public static function load_modules() {
-		if( !self::is_active() && !self::is_development_mode() ) {
-			return;
+		if ( ! self::is_active() && !self::is_development_mode() ) {
+			if ( ! is_multisite() || ! get_site_option( 'jetpack_protect_active' ) ) {
```

Comment:
It'll work just fine.  The point of this code that was preexisting is to short-circuit the module load process if the site is not connected.  This is just saying "if you're not connected, but you happen to be on a multisite install with the `jetpack_protect_active` option, then let's not short-circuit so quickly.  If it's a single site, or that option isn't set, we dump them out just like before.